scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -317,9 +317,9 @@ function Update-IoTEdge {
         # Proxy URI used for all Invoke-WebRequest calls. To specify other proxy-related options like -ProxyCredential, see -InvokeWebRequestParameters
         [Uri] $Proxy,
 
-        # If set to a directory path, the installer prefers to use IoTEdge CAB, Moby Engine CAB, Moby CLI CAB and VC Runtime MSI files from inside this directory
-        # over downloading them from the internet. Thus placing all four files in this directory can be used to have a completely offline install,
-        # or a specific subset can be placed to override the online versions of those specific components.
+        # If set to a directory path, the installer prefers to use IoTEdge CAB and VC Runtime MSI files from inside this directory
+        # over downloading them from the internet. Thus placing both files in this directory can be used to have a completely offline install,
+        # or a specific component's file can be placed to override the online file corresponding to that specific component.
```

Comment:
component (not component's)

English is not native language, but it is my understanding that you use possessive 's only for living things.

Reply:
I'd never heard of such a rule before, and Google suggests it's an old Middle English thing that's largely died out in modern times. Other Google results suggest some Commonwealth nations seem to still preach it (though coming from one of said nations, I can assure you I never heard of such a rule).

Anyway, here's a modern take on it: https://english.stackexchange.com/questions/1031/is-using-the-possessive-s-correct-in-the-cars-antenna tl;dr is that there is no such rule.

Reply:
I'd never heard of such a rule before, and Google suggests it's an old Middle English thing that's largely died out in modern times. Other Google results suggest some Commonwealth nations seem to still preach it (though coming from one of said nations, I can assure you I never heard of such a rule).

Anyway, here's a modern take on it: https://english.stackexchange.com/questions/1031/is-using-the-possessive-s-correct-in-the-cars-antenna tl;dr is that there is no such rule.

Reply:
Woo, some serious language geekiness. This is awesome, thanks Arnav. Now I have an urge to start using thereof :)